db/migrate/create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :phone_number
+      t.integer :role
```

Comment:
It's hard to tell when to draw the line but I think that especially `role` and `phone_number` attributes are a bit too specific to be included in a template for all projects. 
Not all projects use a phone number for users and the project requirements might need a different role/permissions authorization logic and a role column on the user model won't be needed. I know we are trying to avoid boilerplate code with this template but I also don't want to it transform into "bloatware"